### PR TITLE
man: describe how machine-id should be initialized

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/machine-id.xml [hunk 1]
```diff
@@ -53,18 +53,26 @@
...
+    <para>The machine ID is usually generated from a random source during system
+    installation or first boot and stays constant for all subsequent boots. Optionally,
+    for stateless systems, it is generated during runtime during early boot if necessary.
+    </para>
+
+    <para>The machine ID may be set, for example when network booting, with the
+    <varname>systemd.machine_id=</varname> kernel command line parameter or by passing the
+    option <option>--machine-id=</option> to systemd. An ID is specified in this manner
+    has higher priority and will be used instead of the ID stored in
+    <filename>/etc/machine-id</filename>.</para>
```
Comment (poettering) [inline man/machine-id.xml:71]: is that really the case? i remember we used such passed-in uuids only when we /etc/machin-id was so far unset

File: man/machine-id.xml [hunk 2]
```diff
@@ -79,19 +87,58 @@
...
+    <para>After the machine ID is established,
+    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+    will attempt to save it to <filename>/etc/machine-id</filename>. If this fails, it
+    will attempt to bind-mount a temporary file over <filename>/etc/machine-id</filename>.
+    It is an error if the file system is read-only and does not contain a (possibly empty)
+    <filename>/etc/machine-id</filename> file.</para>
+
+    <para><citerefentry><refentrytitle>systemd-machine-id-commit.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    will attempt to write the machine ID to the real file system if
+    <filename>/etc/machine-id</filename> is a mount point.</para>
```
Comment (poettering) [inline man/machine-id.xml]: hmm, this needs a bit different wording i think: "… if /etc/machine-id or /etc are read-only during early boot but become writable later on" or so.

File: man/machine-id.xml (hunk 1 above)
Comment (keszybz) [reply man/machine-id.xml:71]: Yes, I just tested, and it even gets committed to /etc/machine-id.

**Comment by keszybz** [comment]
Updated with the suggested change.

**Pull request merged**